.github/workflows/ci-run-test.yml

```diff
@@ -305,6 +305,7 @@ jobs:
         env:
           PULUMI_NODE_MODULES: ${{ runner.temp }}/opt/pulumi/node_modules
           PULUMI_ROOT: ${{ runner.temp }}/opt/pulumi
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
? Was this not working before

Reply:
This is to try fix the installation error from a remote repo, cc @AaronFriel

Reply:
`403 HTTP error fetching plugin from https://api.github.com/repos/lbrlabs/pulumi-scaleway/releases/tags/v1.2.0` That error? Not sure why it needs a token set at all, that endpoint should be public?

Reply:
The 403 occurs when we're rate limited.

Reply:
Probably, I commented that on https://github.com/pulumi/pulumi/issues/11013. Wonder if we can grab the response body and add that to the error message?

Reply:
So, can this line be reverted?

Reply:
This line can be reverted - it's redundant with lines 59-60 which set the token at global scope:

```yaml
env:
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
I think I understand the issue with these tests. We do send an `Authorization: token $GITHUB_TOKEN` header, but I think the personal access token of the Pulumi bot is unauthorized for that repository.

Reply:
Wonder if we should retry with an empty token when we do the build request, get response pattern here:

https://github.com/pulumi/pulumi/blob/b56aa58d6bf9ad9bad683adb2ee96c10d3c51b2e/sdk/go/common/workspace/plugins.go#L302-L310

(This pattern occurs a few times in plugins.go.)

Reply:
I've reverted this